test/data/sql_injection_postgres/index.php

```diff
@@ -0,0 +1,20 @@
+<html>
+    <body>
+        <a href="/vuln.php?id=1">Article 1</a>
+        <a href="/not_vuln.php?id=2">Article 2</a>
+        <a href="/index.php?id=3">Article 3</a>
+        <a href="/vuln.php">select * from table</a>
+        <a href="/not_vuln.php?id=5">Article 5</a>
+        <a href="/headers_vuln.php">Article 5</a>
```

Comment:
SQL injection should be only in linked page, not on homepage, to check the crawling behavior ;)